docs/setup/linux.md

```diff
@@ -26,7 +26,8 @@ The repository and key can also be installed manually with the following script:
 
 ```bash
 curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
-sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
+sudo apt-key --keyring /etc/apt/trusted.gpg.d/microsoft.gpg add microsoft.gpg
```

Comment:
We were advised by someone from the apt team that apt-key is deprecated and the way we're doing it is the best way right now